lib/guardian_extensions.rb

```diff
@@ -46,8 +50,12 @@ def can_move_chat_messages?(channel)
   end
 
   def can_create_channel_message?(chat_channel)
-    return chat_channel.open? || chat_channel.closed? if is_staff?
-    chat_channel.open?
+    valid_statuses = is_staff? ? %w[open closed] : ["open"]
```

Comment:
Since staff consists of both admins and moderators, I just want to confirm that is what we want here where moderators can create channels and direct message channels as well.

Reply:
Yes that is correct